src/platform/Linux/BUILD.gn

```diff
@@ -76,6 +76,14 @@ static_library("Linux") {
     "${chip_root}/src/setup_payload",
   ]
 
+  cflags = [
+    "-I/usr/include/PCSC",  # Needed for NFCCommissioningManagerImpl
```

Comment:
this should not be global. This has to be based on PCSC dependency. 

Also paths should probably be based on pkgconfig and not just added

Reply:
Please use https://github.com/project-chip/connectedhomeip/blob/master/build/config/linux/pkg_config.gni#L108 and only enable it if `chip_enable_nfc_commissioning`